tests/Query/CombinedFieldsQueryTest.php

```diff
@@ -16,10 +18,10 @@
 class CombinedFieldsQueryTest extends BaseTest
 {
     private static $data = [
-        ['id' => 1, 'title' => 'Rodolfo', 'body' => 'Moraes',   'abstract' => 'Lorem'],
-        ['id' => 2, 'title' => 'Tristan', 'body' => 'Maindron', 'abstract' => 'Dolor'],
-        ['id' => 3, 'title' => 'Monique', 'body' => 'Maindron', 'abstract' => 'Ipsum'],
-        ['id' => 4, 'title' => 'John',    'body' => 'not Doe',  'abstract' => 'Consectetur'],
+        ['id' => '1', 'title' => 'Rodolfo', 'body' => 'Moraes',   'abstract' => 'Lorem'],
```

Comment:
I assume because of line 148, the old syntax would still work?

Reply:
Document id must be string or null. see https://github.com/ruflin/Elastica/blob/8.x/src/Document.php#L45

And it was added long time ago. Clients code must respect this and pass string or null only.